.github/workflows/csharp.yml

```diff
@@ -43,6 +46,7 @@ jobs:
       TRAVIS_COM_TOKEN: ${{ secrets.TRAVIS_COM_TOKEN }}
 
   unit_test:
+    name: Run Unit Tests
     runs-on: ubuntu-latest
```

Comment:
@msohailhussain I'd really like to move this image `runs-on: windows-2022` (like netStandard16) the and ditch the monodevel stuff.

I need to step away from this CI problem and get fresh perspective tomorrow, unless you have ideas.